rules/Php70/EregToPcreTransformer.php

```diff
@@ -189,11 +191,15 @@ private function _ere2pcre(string $content, int $i): array
             throw new InvalidEregException('empty regular expression or branch');
         }
 
-        return [$this->normalize(implode('|', $r)), $i];
+        return [$this->normalize(implode('|', $r), $normalizeUnprintableChar), $i];
     }
 
-    private function normalize(string $content): string
+    private function normalize(string $content, bool $normalizeUnprintableChar): string
     {
+        if ($normalizeUnprintableChar) {
+            $content = str_replace("\x0C", '\\\f', $content);
```

Comment:
On scoper downgrade, it seems replaced with 

```php
        if ($normalizeUnprintableChar) {
            $content = \str_replace("\f", '\\\\f', $content);
        }
```

I will check when it deployed to getrector.com

Reply:
It seems working ok https://getrector.com/demo/b1ed83c8-c2a0-40bb-b3d4-0e098573f462

@bisenegger please try latest `"rector/rector": "dev-main"` and test if the issue persist, better if you can provide reproducible code like the link above, and better, provide patch as well if possible, thank you